### PR TITLE
data keys changes fixed

### PR DESCRIPTION
**Pull request by Kamran95**
## Description
changed keys from camel case to snake case
Fixes #

## Checklist
- [ ] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Run `meteor npm run lint`
- [ ] Added an entry to the `CHANGELOG.md` file.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

**Review by MikeSofaer — COMMENTED** [review]
Why are we changing from sha-256 to sha-1 in package integrity?

**Comment by justyntemme** [comment]
> 
 SHA-1 is a cryptographically broken, we should not be using it for anything. that will allow others to bruteforce our keys and hack our blockchain

File: package-lock.json [hunk 1]
```diff
@@ -2838,302 +2814,624 @@
...
+          "dev": true,
+          "requires": {
+            "is-fullwidth-code-point": "^2.0.0",
+            "strip-ansi": "^4.0.0"
+          },
+          "dependencies": {
+            "strip-ansi": {
+              "version": "4.0.0",
+              "resolved": "https://registry.npmjs.org/strip-ansi/-/strip-ansi-4.0.0.tgz",
+              "integrity": "sha1-qEeQIusaw2iocTibY1JixQXuNo8=",
```
Comment (justyntemme) [inline package-lock.json]: Looks like we are still referencing sha1

**Review by justyntemme — CHANGES_REQUESTED** [review]

File: imports/ui/easel_buy/EaselBuy.jsx [hunk 2]
```diff
@@ -154,21 +156,26 @@ export default class EaselBuy extends Component {
         const nftType = strings.find(
           (val) => val.key.toLowerCase() === "nft_format"
         )?.value;
+        const creator = strings.find(
+          (val) => val.key.toLowerCase() === "creator"
+        )?.value;
 
         const dimentions = this.getNFTDimentions(nftType, itemOutputs);
-        (edition = `${itemOutputs.amountMinted} of ${itemOutputs.quantity}`),
+        (edition = `${itemOutputs.amount_minted} of ${itemOutputs.quantity}`),
```
Comment (justyntemme) [inline imports/ui/easel_buy/EaselBuy.jsx:166]: What is the purpose in switching to snake case?

File: package-lock.json (hunk 1 above)
Comment (faisalnaveedRNS) [reply package-lock.json]: let me have a llok

Comment (faisalnaveedRNS) [reply package-lock.json]: can we have discussion on this , as I did not find any  change in package lock file

Comment (MikeSofaer) [reply package-lock.json]: There is still a change, it is downgrading bootstrap

File: imports/api/recipes/server/methods.js [hunk 3]
```diff
@@ -57,16 +57,16 @@ Meteor.methods({
...
+            recipe.id +
             "&cookbook_id=" +
-            recipe.cookbookID;
+            recipe.cookbook_id;
           recipe.deeplink = deeplink;
           var cookbook_owner = "",
             creator = "";
           try {
-            let cookbooks = Cookbooks.find({ ID: recipe.cookbookID }).fetch();
+            let cookbooks = Cookbooks.find({ ID: recipe.cookbook_id }).fetch();
```
Comment (justyntemme) [inline imports/api/recipes/server/methods.js:67]: Why are we changing conventions?

File: imports/ui/easel_buy/EaselBuy.jsx [hunk 4]
```diff
@@ -115,15 +116,18 @@ export default class EaselBuy extends Component {
         let denom;
         const tradePercent = 100;
         const res = _.cloneDeep(response);
+        console.log("new formateed data is", response);
```
Comment (justyntemme) [inline imports/ui/easel_buy/EaselBuy.jsx]: we can remove this debug line

File: imports/ui/easel_buy/EaselBuy.jsx [hunk 5]
```diff
@@ -222,11 +232,14 @@ export default class EaselBuy extends Component {
...
       media,
+      createdBy,
       displayName,
+      createdAt,
       nftHistory,
+      price,
       denom,
     } = this.state;
-
+    console.log("price is", price);
```
Comment (justyntemme) [inline imports/ui/easel_buy/EaselBuy.jsx]: Remove debug statement

**Review by justyntemme — CHANGES_REQUESTED** [review]
Please remove debug statements and answer questions about convention changes

File: imports/api/recipes/server/methods.js (hunk 3 above)
Comment (faisalnaveedRNS) [reply imports/api/recipes/server/methods.js:67]: chain is changes it convention that why we are changing according to it

File: imports/ui/easel_buy/EaselBuy.jsx (hunk 2 above)
Comment (faisalnaveedRNS) [reply imports/ui/easel_buy/EaselBuy.jsx:166]: chain is changing that why we are changing according to it

File: imports/ui/easel_buy/EaselBuy.jsx (hunk 5 above)
Comment (faisalnaveedRNS) [reply imports/ui/easel_buy/EaselBuy.jsx]: remvoed

File: imports/ui/easel_buy/EaselBuy.jsx (hunk 4 above)
Comment (faisalnaveedRNS) [reply imports/ui/easel_buy/EaselBuy.jsx]: removed

**Review by justyntemme — APPROVED** [review]

**Pull request merged**